feature-libs/order/components/replenishment-order-history/replenishment-order-history.component.ts

```diff
@@ -109,6 +109,7 @@ export class ReplenishmentOrderHistoryComponent implements OnDestroy {
       replenishmentOrderCode
     );
 
+    /* istanbul ignore if */
```

Comment:
Do we really want to ignore code coverage here? it's not our code.

Reply:
I think yes, the same explanation as above